eth/stagedsync/stage_senders_test.go

```diff
@@ -107,7 +108,7 @@ func TestSenders(t *testing.T) {
 
 	require.NoError(stages.SaveStageProgress(tx, stages.Bodies, 3))
 
-	cfg := StageSendersCfg(db, params.TestChainConfig, "")
+	cfg := StageSendersCfg(db, params.TestChainConfig, "", prune.Mode{})
```

Comment:
set real own pls (get from db)

Reply:
That is a test, and as far as stage senders is regarded pruning does not come into place in stage senders since we need senders to still generate the state. I think that putting at least in tests a void prune.Mode would be more clear to read than taking it out of the database.

Reply:
since we do not really use it

Reply:
yep, in test it's ok. 
But then I will ask you - please add 1 test or assert - for new functionality.